### PR TITLE
Description updates

### PR DESCRIPTION
**Pull request by dhaiducek**
Also bumps `go-template-utils` to pick up the description updates for the `template-resolver`.

ref: https://issues.redhat.com/browse/ACM-14630

### CODE REVIEW

File: pkg/policytools/policytools.go [hunk 1]
```diff
@@ -14,11 +14,11 @@ type Cmd struct{}
 func (a Cmd) GetCmd() *cobra.Command {
 	policyCmd := &cobra.Command{
 		Use:   "policytools",
-		Short: "ACM Policy Toolset",
-		Long: `ACM Policy Toolset
+		Short: "RHACM Policy Toolset",
```
Comment (dockerymick) [inline pkg/policytools/policytools.go:17]: RHACM isn't an approved short name

Comment (dhaiducek) [reply pkg/policytools/policytools.go]: Okay--I'll revert the change back to ACM.

Comment (dockerymick) [reply pkg/policytools/policytools.go]: ACM isn't approved either lol it should be Red Hat Advanced Cluster Management. But I saw you used it for the long version

**Review by mprahl — APPROVED** [review]

**Pull request merged**